lib/main_micro_alp.pm

```diff
@@ -424,7 +424,9 @@ sub load_tests {
     } elsif (check_var('EXTRA', 'networking')) {
         load_network_tests;
     } elsif (check_var('EXTRA', 'provisioning')) {
-        loadtest 'microos/verify_setup';
+        # verify_setup is not working correctly in microos with ignition
+        # for the initial configuration. Disabled temporarily to investigate!
+        loadtest 'microos/verify_setup' unless check_var('FIRST_BOOT_CONFIG', 'ignition');
```

Comment:
it cannot affect sle/sle-micro

Reply:
true!  Making changes.

VR:

verify_setup should be present in all except microos with ignition -->

sle-micro with combustion: https://openqa.suse.de/tests/15657610
sle-micro with ignition: https://openqa.suse.de/tests/15657611
microos with combustion: https://openqa.opensuse.org/tests/4552809
microos with ignition: https://openqa.opensuse.org/tests/4552810